layouts/partials/banner.md

```diff
@@ -1,9 +1,9 @@
 {{ if .Params.show_banner -}}
 <div class="o-banner">
 
-The OpenTelemetry tracing specification is now 1.0!
+OpenTelemetry Metrics Release Candidates are now available!
 [Learn more][]
 
-[Learn more]: https://medium.com/opentelemetry/opentelemetry-specification-v1-0-0-tracing-edition-72dd08936978
+[Learn more]: https://opentelemetry.io/blog/2022/metrics-announcement/
```

Comment:
It has been like that before, so only as optional change: There are 2 "Learn More" on that page, although web accessibility suggest to [ensure that links with the same accessible name serve a similar purpose](https://dequeuniversity.com/rules/axe/4.2/identical-links-same-purpose)